src/components/Messages/PreviewList.tsx

```diff
@@ -62,17 +65,41 @@ const PreviewList: FC = () => {
   };
 
   return (
-    <GridItemFour className="sm:h-[76vh] md:h-[80vh] xl:h-[84vh]  ">
-      <Card className="h-full">
+    <GridItemFour className="sm:h-[76vh] md:h-[80vh] xl:h-[84vh] mb-0">
+      <Card className="h-full flex justify-between flex-col">
         <div className="flex justify-between items-center p-5 border-b">
           <div className="font-bold">Messages</div>
-          <button onClick={newMessageClick} type="button">
-            <PlusCircleIcon className="h-6 w-6" />
-          </button>
+          {currentProfile && !showAuthenticating && !showLoading && (
+            <button onClick={newMessageClick} type="button">
+              <PlusCircleIcon className="h-6 w-6" />
+            </button>
+          )}
         </div>
-        <div className="mt-2">
-          {showLoading ? (
-            <PageLoading message="Loading conversations" />
+        <div className="h-full">
+          {showAuthenticating ? (
+            <div className="flex h-full justify-center items-center">
+              <PageLoading message="Awaiting signature to enable DMs" />
+            </div>
+          ) : showLoading ? (
+            <div className="flex h-full justify-center items-center">
+              <PageLoading message="Loading conversations" />
+            </div>
+          ) : !currentProfile ? (
+            <div className="flex h-full justify-center items-center">
+              <EmptyState
+                message={<div>Login to start messaging</div>}
+                icon={<MailIcon className="w-8 h-8 text-brand" />}
+                hideCard
+              />
+            </div>
```

Comment:
**suggestion:** this check is not required 🙏🏼 cuz we return 404 in higher level

Reply:
Hmm but we need to have it at page level because DM UI is protected and should not be visible to public 🙏🏼

Reply:
Updated the signed out screenshot in the description of the PR to demonstrate. Should be all set!

Reply:
Amazing 🚀